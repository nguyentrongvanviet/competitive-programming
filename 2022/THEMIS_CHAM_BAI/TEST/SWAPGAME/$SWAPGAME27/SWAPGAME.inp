110
77 50 9 81 35 59 101 71 55 80 49 73 32 44 16 84 79 60 12 83 56 40 81 30 102 34 18 14 88 108 2 33 87 57 19 78 20 23 49 99 76 21 6 51 45 86 60 14 57 28 58 50 18 31 100 9 94 26 52 11 75 17 54 41 69 15 78 62 64 53 85 65 47 33 107 96 106 67 37 42 80 7 42 108 61 39 4 93 4 83 109 12 8 106 5 86 8 90 97 76 59 24 40 43 77 7 48 103 32 29 
72 68 99 66 61 107 101 71 39 41 48 25 46 38 94 28 90 110 22 3 98 104 79 30 102 27 95 13 24 89 2 58 87 74 54 35 17 23 63 10 92 21 15 73 70 91 110 53 74 84 69 25 95 31 100 13 16 26 105 64 75 20 19 37 97 6 98 62 11 47 5 65 43 63 66 67 34 96 68 56 51 82 52 89 38 55 109 93 10 3 44 22 36 27 85 91 36 103 45 92 105 88 104 1 70 82 72 1 46 29 