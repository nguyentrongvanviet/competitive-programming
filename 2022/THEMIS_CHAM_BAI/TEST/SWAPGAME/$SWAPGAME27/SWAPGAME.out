67108864
34